### ISSUE TITLE
[BUG] Login issues with David Walsh method enabled

### ISSUE THREAD

**Issue by EusebiuOprinoiu** [Highfivery/wordpress-zero-spam#310]
Hello, Ben!

After the last update, all login attempts fail via the WooCommerce login form. 
The issue is caused by the David Walsh protection method.

**Comment by bmarshall511** [comment]
@EusebiuOprinoiu Please follow the instructions when submitting an issue, I need more details than 'it doesn't work' to track down what's going on. Can you provide an example URL?

**Comment by EusebiuOprinoiu** [comment]
Hello, Ben!

I can't provide an example URL. I don't have any WooCommerce websites.
The issue was brought to my attention by a friend that uses WooCommerce, and right now, he has Zero Spam disabled.
He is in the middle of a product launch and he won't re-enable your plugin until the issue is fixed. (he received multiple emails and phone calls from customers that could not log in to their accounts)
I just helped him debug the issue.

As for additional details, I don't know what else I can provide. All login attempts via the WooCommerce form fail when the David Walsh protection method is enabled. (on every OS, every browser, every device)

Anyway, I checked your code, and I noticed that on your last update you added support for WooCommerce registrations. But while doing so, you removed (presumably by mistake) the selector for the WooCommerce login form.

In the `modules/davidwalsh/assets/js/davidwalsh.js` file, your list of selectors should include `.woocommerce-form-login` as well. Right now only `.woocommerce-form-register` is on that list.

The complete list of selectors should be:
```var selectors = "#commentform, #registerform, .wpforms-form, .wpcf7-form, .frm-fluent-form, #loginform, .mepr-signup-form, .mc4wp-form, #mepr_loginform, .woocommerce-form-login, .woocommerce-form-register";```
